### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by johnsonw**
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2395)

### CODE REVIEW

**Comment by johnsonw** [comment]
Service Test output (from a local run):

```
+ PYTHONPATH=.
+ nosetests tests/services/ --stop
/usr/lib64/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
/usr/lib/python2.7/site-packages/paramiko/transport.py:33: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in a future release.
  from cryptography.hazmat.backends import default_backend
.........................../usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
../usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
127.0.0.1 - - [01/Dec/2020 15:10:51] "POST /agent/register/ HTTP/1.0" 200 -
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
127.0.0.1 - - [01/Dec/2020 15:12:22] "POST /agent/message/ HTTP/1.0" 200 -
/usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
/usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
127.0.0.1 - - [01/Dec/2020 15:12:22] "GET /agent/message/ HTTP/1.0" 200 -
/usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
127.0.0.1 - - [01/Dec/2020 15:12:23] "POST /agent/reregister/ HTTP/1.0" 200 -
./usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:843: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
...
----------------------------------------------------------------------
Ran 38 tests in 1082.616s

OK
```

File: tests/framework/services/runner.sh.in [hunk 1]
```diff
@@ -71,12 +72,17 @@ pip install -r requirements.txt
 pip install -r requirements.test
 echo -e "/^DEBUG =/s/= .*$/= True/\nwq" | ed settings.py
 
-cp -r /integrated-manager-for-lustre /usr/share/chroma-manager
+rsync -ar --exclude='.cargo' --exclude='iml-gui/node_modules' /integrated-manager-for-lustre/ /usr/share/chroma-manager
 mkdir /var/log/chroma
+mkdir /var/lib/chroma
```
Comment (jgrund) [inline tests/framework/services/runner.sh.in:77]: This should happen in dev_setup

File: tests/unit/chroma_core/models/test_advertised_jobs.py [hunk 2]
```diff
@@ -279,11 +282,72 @@ def create_fake_filesystem_client(self, active=False):
         from chroma_core.models import ManagedMgs, ManagedMdt, ManagedOst, ManagedFilesystem, LustreClientMount
         from tests.unit.chroma_core.helpers import synthetic_volume_full
 
-        mgt, _ = ManagedMgs.create_for_volume(synthetic_volume_full(self.server).id, name="MGS")
-        fs = ManagedFilesystem.objects.create(mgs=mgt, name="testfs")
+        # mgt, _ = ManagedMgs.create_for_volume(synthetic_volume_full(self.server).id, name="MGS")
+        # fs = ManagedFilesystem.objects.create(mgs=mgt, name="testfs")
+        # ObjectCache.add(ManagedFilesystem, fs)
+        # ManagedMdt.create_for_volume(synthetic_volume_full(self.server).id, filesystem=fs)
+        # ManagedOst.create_for_volume(synthetic_volume_full(self.server).id, filesystem=fs)
```
Comment (jgrund) [inline tests/unit/chroma_core/models/test_advertised_jobs.py:289]: cruft

**Comment by jgrund** [comment]
Just a few small comments.

**Review by jgrund — DISMISSED** [review]

**Review by ip1981 — DISMISSED** [review]

**Review by jgrund — APPROVED** [review]

**Pull request merged**